docs/extensions/effect.mdx

```diff
@@ -110,6 +113,25 @@ function MyComponent() {
   })
   ```
 
+- **Supports Recursion:**
+  Recursion is supported with `set.recurse` for both sync and async use cases, but is not supported in the cleanup function.
+
+  <!-- prettier-ignore -->
```

Comment:
What happens without it?
I think it's effective until the end. Is there a way to enable it back?

Reply:
I'm sorry, I don't understand what you are asking.

Reply:
If you are asking about prettier, prettier breaks the html layout for details arrows.

Reply:
Yes, I was asking about prettier. Maybe give up `<details>` for now?

Reply:
> Yes, I was asking about prettier. Maybe give up `<details>` for now?

I think the reader experience is good by initially hiding this additional information.

HTML elements in MDX is valid. The `<!-- prettier-ignore -->` only affects the next line. If removed, prettier adjusts the line spacing and indentation until MDX readers no longer parse this as HTML. Its too bad prettier doesn't have good support for this.

Reply:
> The `<!-- prettier-ignore -->` only affects the next line.

Oh, I see!